src/js/components/atoms/c-image/c-image.ts

```diff
@@ -56,15 +55,27 @@ export default class CImage extends Mixins(InView) implements ICImageProps {
 
         await this.calculateDimensions(true);
         this.ready = true;
-
-        this.$el.addEventListener('load', this.thumbnailLoadHandler);
     }
 
     beforeDestroy() {
         this.viewportUtil.removeResizeHandler(this.resizeHandler);
         this.viewportUtil.removeScrollHandler(this.resizeHandler);
     }
 
+    @Watch('url')
+    async urlChanged() {
+        // replaces previous image with tinyGif
```

Comment:
Code should explain what it does, comments should not.

Reply:
I wanted to take a look at this outside of this feature request.